src/query/execute_schema.c

```diff
@@ -10610,7 +10610,6 @@ do_change_att_schema_only (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NOD
     {
       MOP auto_increment_obj = NULL;
 
-      assert (attribute->info.attr_def.auto_increment != NULL);
```

Comment:
Can assert be deleted? What test cases can I check this with?

Reply:
it can raise segement fault at debug mode if the auto_increment is not figured. So, I remove the assert.